iacrcc/iacrcc.cls

```diff
@@ -1080,6 +1075,19 @@
   }{}% final
 }
 
+% Redefine the abstract environment to write it to the .abstract file
+\renewenvironment{abstract}{%
+  \filecontents[nosearch,noheader,overwrite]{\jobname.abstract}%
```

Comment:
I think we should remove the `noheader` option for several reasons:
* This makes it clear to the author who created the `\jobname.abstract` file,
* It can be useful for debugging in the server.
* the server strips out comments from the abstract anyway.

The header would look like:
```
%% LaTeX2e file `./main.abstract'
%% generated by the `abstract' environment
%% from source `main' on 2023/05/26.
%%
```

Reply:
Agreed, I will let filecontents create these header comments.